### DISCUSSION TITLE
Enhancement: Add plugin Support to ANTHROPIC

### DISCUSSION THREAD

**Discussion by ido777** [danny-avila/LibreChat#2428] [Feature Requests & Suggestions]
### What features would you like to see added?

ANTHROPIC support tools in Beta - it will be great to have a plugin for them as well 
https://docs.anthropic.com/claude/docs/tool-use

### More details

Maybe it should be more generic so that it won't call for code changes once more providers support tools

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MchLrnX** [comment]
Yes please

**Comment by rickykresslein** [comment]
I would much prefer to use the Web Search plugin with Claude rather than ChatGPT. Is there any plan to implement this?

**Reply by shaggy2626** [reply]
Same here, i only use Claude and cant use my plugins anymore because im limited to openai only.

**Comment by danny-avila** [comment]
Being addressed by: https://github.com/danny-avila/LibreChat/issues/3607

The feature is currently available in experimental stage
